src/ui/public/paginated_table/paginated_table.html

```diff
@@ -10,6 +10,8 @@
           <th
             ng-repeat="col in columns"
             ng-click="paginatedTable.sortColumn($index)"
+            kbn-accessible-click
+            tabindex="0"
```

Comment:
Awesome! @snide do you think you could fix the focus styles for these elements?

![image](https://user-images.githubusercontent.com/1238659/29078462-5971f2be-7c0f-11e7-8a0f-ebaa371d1bef.png)